CRM/Contact/BAO/Contact/Permission.php

```diff
@@ -60,13 +158,16 @@ public static function allow($id, $type = CRM_Core_Permission::VIEW) {
       return TRUE;
     }
 
-    //check permission based on relationship, CRM-2963
+    // check permission based on relationship, CRM-2963
```

Comment:
While we are being fussy it's supposed to be a capital letter & a full stop at the end....